tasks/main.yml

```diff
@@ -5,14 +5,14 @@
 - name: test to see if selinux is running
   command: getenforce
   register: sestatus
-  changed_when: false
-  ignore_errors: true
+  failed_when: false
+  always_run: true
```

Comment:
Ah, ok, I think I got it. Your goal is to see next tasks actions in dry_mode. My proposal will left them unchanged even if in normal mode they will be changed. let me think